src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -1,4 +1,5 @@
 @import '../../../common/variables';
+$container-padding: 25px;
```

Comment:
This will be declared as a global variable. Please move it inside its most relevant style rule, such as `.bcs-activity-feed-app-activity`, to make it local instead.